### DISCUSSION TITLE
If a linked data model will be provided for open-data-product-spec-2.1 ?

### DISCUSSION THREAD

**Discussion by xdxxxdx** [Open-Data-Product-Initiative/v3.0#65] [General]
https://open-data-product-initiative.github.io/open-data-product-spec-2.1/#specification-aims-and-aspects
By following the spec, I am wondering if there is a smart data model for  open-data-product-spec-2.1? I mean data model with linked data??
Thanks

**Comment by kyyberi** [comment]
We have had discussions of linked data model as well, but so far nothing for that has been done obviously. If you would like to have it, please add a feature request to the Dev version (https://github.com/Open-Data-Product-Initiative/dev/issues) and we can start practical exploration how to add that in the spec

**Reply by xdxxxdx** [reply]
> We have had discussions of linked data model as well, but so far nothing for that has been done obviously. If you would like to have it, please add a feature request to the Dev version (https://github.com/Open-Data-Product-Initiative/dev/issues) and we can start practical exploration how to add that in the spec

Hello @kyyberi , thanks for reply, the feature request is here: https://github.com/Open-Data-Product-Initiative/dev/issues/12
